code/config/config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "development": {
```

Comment:
Im not sure what this file does, I believe it is generated automatically when we make migrations